api/v1alpha1/types.go

```diff
@@ -58,7 +58,7 @@ type ImageBasedUpgradeSpec struct {
 	Stage            ImageBasedUpgradeStage `json:"stage,omitempty"`
 	SeedImageRef     SeedImageRef           `json:"seedImageRef,omitempty"`
 	AdditionalImages ConfigMapRef           `json:"additionalImages,omitempty"`
-	OADPContent      ConfigMapRef           `json:"oadpContent,omitempty"`
+	OADPContent      []ConfigMapRef         `json:"oadpContent,omitempty"`
```

Comment:
We couldn't directly import oadp module due to a version conflict issue between controller-runtime version used in velero and the one used in lifecycle-agent.  Since the oadp module not only imports its API but also launches the velero server, the version conflict causes lifecycle agent fail to start. 
I would suggest keeping the OADP naming, as from user's perspective, everything is controllered/configured via OADP.